lib/AI-Agent.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <ApplicationInsightsAgent>
-    <AgentLogger>INFO</AgentLogger>
+    <AgentLogger>DEBUG</AgentLogger>
     <Instrumentation>
         <BuiltIn enabled="true">
```

Comment:
was just wondering if it's simpler to just log the user info at INFO level. Less changes would be required.